README.md

```diff
@@ -47,6 +47,7 @@ The following table lists the most recent few versions of the operator.
 | Operator Version | API Version | Kubernetes Version | Base Spark Version | Operator Image Tag |
 | ------------- | ------------- | ------------- | ------------- | ------------- |
 | `latest` (master HEAD) | `v1beta2` | 1.13+ | `3.0.0` | `latest` |
+| `v1beta2-1.2.3-3.1.1` | `v1beta2` | 1.13+ | `3.1.1` | `v1beta2-1.2.3-3.1.1` |
```

Comment:
`v1beta2-1.2.1-3.1.1`?

Reply:
There was already release [v1beta2-1.2.2](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/releases/tag/v1beta2-1.2.2-3.0.0) so next one should be v1beta2-1.2.3, right?

Reply:
Yes, `v1beta2-1.2.3-3.1.1` is fine.